### PR TITLE
Add Language translation for Indonesian

### PR DESCRIPTION
**Pull request by khairu-aqsara**
## Description

add translation support in Indonesian

### CODE REVIEW

File: app/locales/id.json [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+      "authorities_removal_alert_desc": "Apakah anda yakin akan menghapus sumber Otoritas kesehatan?",
+      "authorities_removal_alert_proceed": "Diproses",
+      "authorities_removal_alert_title": "Hapus Otiritas kesehatan",
+      "authorities_title": "Sumber Terpercaya",
+      "choose_provider_subtitle": "Untuk mendapat informasi tentang paparan, Anda harus berlangganan ke otoritas kesehatan.",
+      "choose_provider_title": "Pilih Otoritas kesehatan",
+      "commitment": "Komitmen",
+      "commitment_para": "Safe Paths dengan aman mencatat dan memeriksa interaksi Anda dengan orang-orang yang menggunakan lokasi Anda. Data Anda TIDAK AKAN PERNAH dikirimkan dan digunakan tanpa persetujuan Anda.",
+      "default_news_site_name": "Berita Safe Paths",
+      "default_news_site_url": "https://covidsafepaths.org/in-app-news",
```
Comment (tstirrat) [inline app/locales/id.json:19]: Remove this

File: app/locales/id.json [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+      "event_history_title": "Riwayat paparan",
+      "export_para_1": "Jika Anda dinyatakan positif COVID-19, Bertanggung Jawablah dengan membagikan riwayat lokasi Anda dengan otoritas kesehatan.",
+      "export_para_2": "Lokasi yang dibagikan berupa tempat dan waktu, tidak ada informasi tambahan yang lain",
+      "home_at_risk_header": "Anda Mungkin Terpapar",
+      "home_at_risk_subsubtext": "Ini tidak berarti Anda terinfeksi.",
+      "home_at_risk_subtext": "Berdasarkan riwayat GPS Anda, ada kemungkinan Anda melakukan kontak atau dekat dengan seseorang yang didiagnosis dengan COVID19.",
+      "home_enable_location": "Aktifkan Data Lokasi",
+      "home_mayo_link_heading": "Informasi COVID-19",
+      "home_mayo_link_label": "Dari Mayo Clinic",
+      "home_mayo_link_URL": "https://www.mayoclinic.org/coronavirus-covid-19",
```
Comment (tstirrat) [inline app/locales/id.json:30]: Remove this line

File: app/locales/id.json [hunk 3]
```diff
@@ -0,0 +1,112 @@
...
+      "push_at_risk_message": "Anda telah berpapasan dengan pasien COVID-19",
+      "push_at_risk_title": "Anda mungkin berisiko",
+      "see_exposure_history": "Lihat riwayat paparan",
+      "settings_title": "Dashboard",
+      "share_location_data": "Bagikan riwayat lokasi",
+      "show_overlap": "Click untuk melihat data publik",
+      "team": "Tim",
+      "team_para": "Tim kami terdiri dari konsorsium ahli epidemiologi, insinyur, ilmuwan, ahli privasi digital, profesor dan peneliti dari lembaga terkemuka, termasuk: MIT, Harvard, Klinik Mayo, TripleBlind, EyeNetra, Ernst & Young dan Link Ventures.",
+      "terms_of_use": "Syarat Penggunaan",
+      "terms_of_use_url": "https://docs.google.com/document/d/1mtdal_pywsKZVMXLHjjj5eKznipPLP8sM1HwFTIhjo0/edit#",
```
Comment (tstirrat) [inline app/locales/id.json:98]: Remove this too

File: app/locales/languages.js [hunk 4]
```diff
@@ -77,6 +78,7 @@ i18next.init({
     es: { label: 'Español', translation: es },
     fr: { label: 'Français', translation: fr },
     ht: { label: 'Kreyòl ayisyen', translation: ht },
+    id: { label: 'Indonesia', translation: id },
```
Comment (tstirrat) [inline app/locales/languages.js:81]: Please verify if this is correct for the dropdown list

Comment (alpita-masurkar) [reply app/locales/languages.js:81]: id for Indonesian is correct

Comment (tstirrat) [reply app/locales/languages.js:81]: Thanks

File: app/locales/id.json [hunk 5]
```diff
@@ -0,0 +1,110 @@
...
+    "event_history_title": "Riwayat paparan",
+    "export_para_1": "Jika Anda dinyatakan positif COVID-19, Bertanggung Jawablah dengan membagikan riwayat lokasi Anda dengan otoritas kesehatan.",
+    "export_para_2": "Lokasi yang dibagikan berupa tempat dan waktu, tidak ada informasi tambahan yang lain",
+    "home_at_risk_header": "Anda Mungkin Terpapar",
+    "home_at_risk_subsubtext": "Ini tidak berarti Anda terinfeksi.",
+    "home_at_risk_subtext": "Berdasarkan riwayat GPS Anda, ada kemungkinan Anda melakukan kontak atau dekat dengan seseorang yang didiagnosis dengan COVID19.",
+    "home_enable_location": "Aktifkan Data Lokasi",
+    "home_mayo_link_heading": "Informasi COVID-19",
+    "home_mayo_link_label": "Dari Mayo Clinic",
+    "home_mayo_link_URL": "https://www.mayoclinic.org/coronavirus-covid-19",
```
Comment (alpita-masurkar) [inline app/locales/id.json:29]: do we want this URL key here or should it be removed @tstirrat

**Comment by alpita-masurkar** [comment]
hi @khairu-aqsara, I was wondering if you're available. There's been updates to the English content on the app and we need to make updates to Indonesian. Looking forward to hearing from you.

**Pull request merged**